pgmpy/inference/CausalInference.py

```diff
@@ -97,6 +97,8 @@ def is_valid_backdoor_adjustment_set(self, X, Y, Z=[]):
         >>> inference.is_valid_backdoor_adjustment_set("X", "Y")
         True
         """
+        if isinstance(Z, str):
+            Z = [Z]
         Z_ = list(Z)
```

Comment:
@1betatsu Thanks for the PR. Could possibly make this a bit cleaner by replacing this line with something like: `Z_ = [Z] if isinstance(Z, str) else list(Z))`. Also could you please add a test for this change?

Reply:
@ankurankan Thank you for your quick feedback!
I would add tests to validate Z of is_valid_backdoor_adjustment_set. I also forgot formatting by black. I would fix it, too.